client/my-sites/checkout/checkout-thank-you/marketplace/use-themes-thank-you-data.tsx

```diff
@@ -90,12 +84,29 @@ export function useThemesThankYouData( themeSlugs: string[] ): ThankYouData {
 	);
 	const isAtomicNeeded = hasDotOrgThemes || hasExternallyManagedThemes;
 
+	const hasFreeThemes = themesList.some( ( theme ) => theme?.cost?.number === 0 );
```

Comment:
Is there a better way to do this or will this do?

Reply:
This is returning `false` when there is a free theme selected, I tried it out with the free theme storia 🤔 .

Reply:
I tried using `canUseTheme` to check instead. Could you also help test it out to see if it works as intended?